README.md

```diff
@@ -1,7 +1,7 @@
 # Fast Forward Pull Requests
 
 This repository contains a GitHub action that merges a pull request by
-fast forwarding the target branch.  The action is triggered when an
+fast forwarding the target branch. The action is triggered when an
```

Comment:
I'm a bit confused about this and other similar changes.  It seems that this is doing the opposite of what the commit message says: instead of fixing the double spaces after full stops, it is removing them!